rosetta-source/src/main/rosetta/event-common-func.rosetta

```diff
@@ -1824,7 +1830,81 @@ func Create_TerminationInstruction: <"Creates the relevant primitive instruction
 	set instruction -> quantityChange: <"Set primitive instruction to be only a quantity change instruction (with values set to zero).">
 		Create_QuantityChangeInstruction( [changePriceQuantity], QuantityChangeDirectionEnum->Replace, empty )
 
-// End of ICMA - Phase 2 Contribution
+func Create_PartialDeliveryPrimitiveInstruction: <"Creates the primitive instructions for a substitution of collateral by replacing the assetpayout of the trade.">
+	inputs:
+		tradeState TradeState (1..1) <"The original trade to be closed.">
+		effectiveDate AdjustableOrRelativeDate (1..1) <"The date to close and open a new position.">
+		terminationDate AdjustableOrRelativeDate (1..1) <"The new termination date.">
+		deliveredPriceQuantity PriceQuantity (1..*) <"The price and quantity of the remaining trade. ">
```

Comment:
think the comment for deliveryPriceQuantity should be "The price and quantity to set on the original trade" as this is the amount that has actually been delivered